.github/workflows/build.yml

```diff
@@ -101,6 +101,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81 # v5.5.1
         with:
+          context: ${{ inputs.tag != '' && 'git' || 'workflow' }}
```

Comment:
What is this doing?

Reply:
That action by default that action uses the workflow context (https://docs.github.com/en/actions/learn-github-actions/contexts#github-context).

During a normal build it's fine because workflow context has the same info as the commit, but when we rebuild an image we want to get the info (like the commit) from the tag that was checked out.